hbw/app/javascript/packs/hbw/components/container.js.jsx

```diff
@@ -24,6 +24,46 @@ modulejs.define(
         return tasks.filter(task => task.entity_code === entityCode).reverse();
       };
 
+      getTasksFormsByList = async (tasks) => {
+        const emptyFormTasks = this.getEmptyFormEntityTasks(tasks);
+        const { connection } = this.props.env;
+        const { addForms } = this.context;
+
+        if (emptyFormTasks.length > 0) {
+          const forms = await connection.request({
```

Comment:
If you will use destruction here, then you can avoid  forms.forms in  mergeFormsToTasks